HW35_SlotMachineGame.py

```diff
@@ -0,0 +1,56 @@
+"""
+  Slot machine game
+"""
+class SlotMachineGame:
+
+    def __init__(self, balance: int, isMember = None, freeGame = True):
+        self.balance = balance
+        self.isMember = isMember
+        self.freeGame = freeGame
+
+    WIN = 1000000
+    GAME_FEE = 50
+
+    def _spin(self):
+        """
+                Game fee deducts from balance,
+                if two random numbers match it's a win, balance increases with the WIN amount
+        """
+        import random
+
+        self.balance -= SlotMachineGame.GAME_FEE
+        if random.random() == random.random():
+            self.balance += SlotMachineGame.WIN
+            print(f"You won {SlotMachineGame.WIN}!!! Your budget now is {self.balance}!!!")
+        else:
+            print("You lost! Try again!")
+
+    def play(self):
+        """
+        If balance amount is sufficient game starts, if not - top up is required
+        Members are allowed to play once for free if broke
```

Comment:
I would make self.freeGame private or protected as it shouldn't be changed outside